packages/integrations/sitemap/src/index.ts

```diff
@@ -133,7 +133,10 @@ const createPlugin = (options?: SitemapOptions): AstroIntegration => {
 						return urls;
 					}, []);
 
-					pageUrls = Array.from(new Set([...pageUrls, ...routeUrls, ...(customPages ?? [])]));
+					const customPagesUrls =
```

Comment:
Here, `customPages(config.site)` can return `undefined`, which is something we should check

Reply:
Line 139 already does that with the nullish coalescing operator